packages/core/src/tools/observable.ts

```diff
@@ -18,3 +20,5 @@ export class Observable<T> {
     this.observers.forEach((observer) => observer(data))
   }
 }
+
+export type ErrorObservable = Observable<RawError>
```

Comment:
I'm not sure this type has its place there. I think, either:
* put it in a new `domain/error/types.ts`
* put it in `tools/error`
* replace `ErrorObservable` with `Observable<RawError>` everywhere

Reply:
I would simply use `Observable<RawError>` because it's already used in some places

Reply:
Yep, using Observable<RawError> is probably easier and still well express the intent.